content/300-guides/200-deployment/110-deployment-guides/400-deploying-to-aws-lambda.mdx

```diff
@@ -86,181 +88,115 @@ schema.sql
-  title     String
-  content   String?
-  published Boolean  @default(false)
-  User      User     @relation(fields: [authorId], references: [id]) // relation field
-  authorId  Int // relation scalar field
-}
-
-model Profile {
-  id     Int     @id @default(autoincrement())
-  bio    String?
-  userId Int     @unique // relation scalar field
-  User   User    @relation(fields: [userId], references: [id]) // relation field
-}
+Open Prisma Studio:
 
-model User {
-  id      Int      @id @default(autoincrement())
-  email   String   @unique
-  name    String?
-  Post    Post[] // relation field
-  Profile Profile? // relation field
-}
+```terminal
+npx prisma studio
 ```
 
-You can now run `prisma generate` to generate the Prisma Client based on your new Prisma schema. Whenever you make changes to your Prisma schema in the future, you manually need to invoke `prisma generate` in order to accommodate the changes in your Prisma Client API.
+Your database has three models: `Post`, `Profile`, and `User`.
 
-### Rename the relation fields for easy access
+## 4. Set your AWS Access key as an environment variable
 
-Because both the generated `Post` and `Profile` fields in the `User` model are _virtual_ (i.e. they're _not backed by a foreign key in the database_), you can manually rename them in your Prisma schema. This will only affect the generated client and is typically done so that they have a more meaningful name in the context of the relation.
+In order for the Serverless Framework to provision the AWS resources and deploy your application, you must configure your access key.
 
-In the resulting Prisma schema there are two types of relation fields:
+The method you use to get the access key depends on whether you use your personal account or create a special IAM user for the Serverless Framework. We recommend that you create a special IAM user because it lets you set granular permissions. To get an access key for your account, follow the [AWS guide](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey)
 
-- Relation fields: identified by having a Model name as the type, e.g. the `User` field in the `Post` model. Can be renamed to better fit its usage, e.g. `User` -> `author`.
-- [Relation scalar fields](/concepts/components/prisma-schema/relations): these are used to store the foreign key, e.g. the `authorId` field in the `Post` model. Cannot be rename as it must match the field in the database.
+When you have the _Access key ID_ and a _Secret access key_, set them with the following command. In the following command, replace the placeholders with your _Access key ID_ and _Secret access key_.
 
-The names of the relation fields are used in the client to access those relations for example, fetching a specific `Post` and its associated `User` object would as follows with the Prisma schema above:
+If you installed the Serverless Framework CLI globally, then use the following command:
```

Comment:
I've just noticed that you're giving both options, with and without `npx`. I have 2 options on how to improve this:

1. Use [Tabbed blocks](https://www.prisma.io/docs/about/prisma-docs/docs-components/mdx-examples#tabbed-blocks).
2. Provide this explanation in the Prerequisites section at the beginning explaining the difference between global and local project installation. And describe that the guide uses the global or the local project installation and all provided commands are either `serverless ...` or `npx serverless ...`.

My personal preference would be for option 2 as that would make it easier to digest the info without having to think about tabs and their markup.